interface/main/left_nav.php

```diff
@@ -1118,6 +1208,23 @@ function selpopup(selobj) {
 
     </ul>
   </li>
+  <?php if($GLOBALS['enable_group_therapy']) : ?>
+      <li class="open"><a class="expanded" id="patimg" ><i class="fa fa-fw fa-users"></i>&nbsp;<span><?php xl('Group','e') ?></span></a>
+          <ul>
+              <?php genMiscLink('RTop','gfn','0',xl('Groups'),'therapy_groups/index.php?method=listGroups'); ?>
+              <?php genTreeLink('RTop','gng', xl('New')); ?>
+              <?php genTreeLink('RTop','gdg',xl('Group Details')); ?>
+              <li class="open"><a class="expanded_lv2"><span><?php xl('Visits','e') ?></span></a>
+                  <ul>
+                      <?php genTreeLink('RBot','gcv',xl('Create Visit')); ?>
+                      <?php genTreeLink('RBot','gce',xl('Current')); ?>
+                      <?php genTreeLink('RBot','gvh',xl('Visit History')); ?>
+                  </ul>
+              </li>
+
+          </ul>
+      </li>
+  <?php endif ?>
   <?php // TajEmo Work by CB 2012/06/21 10:41:15 AM hides fees if disabled in globals ?>
   <?php if(!isset($GLOBALS['enable_fees_in_left_menu']) || $GLOBALS['enable_fees_in_left_menu'] == 1){ ?>
   <li><a class="collapsed" id="feeimg" ><span><?php xl('Fees','e') ?></span></a>
```

Comment:
Would it be possible for the tabs developer to have a go at it?
We are under a lot pf pressure here and didn't take into account we would have to integrate this.